src/jltypes.c

```diff
@@ -3610,6 +3610,9 @@ void jl_init_types(void) JL_GC_DISABLED
     jl_value_t *listt = jl_new_struct(jl_uniontype_type, jl_task_type, jl_nothing_type);
     jl_svecset(jl_task_type->types, 0, listt);
 
+    const static uint32_t task_constfields[1] = {0x00000040}; // Set fields 7 as constant
```

Comment:
@vtjnash should any of the task fields be atomic? I was surprised that things like `_state` weren't.

Reply:
Most likely yes, correctness would seem to demand that (esp for acquire/release of result after the _state read/write). I think it must have gotten missed in my list of data race issues